src/codeql.test.ts

```diff
@@ -20,14 +21,18 @@ import {
   getQueryPackInfo,
 } from "./codeql";
 
-const test = anyTest as TestFn<{ db: string; tmpDir: string }>;
+const test = anyTest as TestFn<{
```

Comment:
This will conflict with https://github.com/github/codeql-variant-analysis-action/pull/988 but should be easy to fix up